### DISCUSSION TITLE
Option to disable Move Left/Move Right depending on the pane you are on.

### DISCUSSION THREAD

**Discussion by KevinMatney** [WinMerge/winmerge#664] [Q&A] [answered]
Hello - is there a way for the right-click menu to only have one "Copy to Left" or "Copy to Right" enabled, depending on the pane you are on.  The old versions only gave you one choice, you copied the code you wanted to keep to the other pane.  With the latest versions, it is very easy to accidently click on the wrong selection, which of course will do the opposite of what you want.  I now have to be very careful about right-clicking, and makes the software harder to use.  I want the lines I right click on top be the lines I want to keep, and switch panes as needed.  To easy to make a mistake now.

**Comment by sdottaka** [comment, marked as answer]
Make one file Read-only. In WinMerge 2.16.10, you can make it Read-only by double-clicking the status bar as shown below.
![RO](https://user-images.githubusercontent.com/98126/110208708-1e32b580-7ecc-11eb-8f0e-d09d3d852a4f.gif)

**Comment by KevinMatney** [comment]
Thanks for the reply.  The way I use Winmerge is that I merge changes both left and right until the files are identical - that way I know I've addressed all changes.  So this work-around is a bit inconvenient for my work flow.  For now, I'm just sticking with older versions on Winmerge - version 2.14 doesn't have this issue.  With the latest version a mis-click cost me hours of time...
